internal/cli/atlas/integrations/create/datadog.go

```diff
@@ -69,8 +70,15 @@ func DatadogBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     datadogType,
 		Aliases: []string{"datadog"},
-		Short:   "Create or update the Datadog integration.",
-		Args:    require.NoArgs,
+		Short:   "Create or update a Datadog integration for your project.",
+		Long: `The requesting API key must have the Organization Owner or Project Owner role to configure an integration with Datadog.
+
+After you integrate with Datadog, you can send metric data about your project to your Datadog dashboard. To learn more about the metrics available to Datadog, see https://www.mongodb.com/docs/atlas/tutorial/datadog-integration/.
+		
+Datadog integration is available only on M10+ clusters.`,
```

Comment:
Suggestion: how about "supported only on" or "available only for":
```suggestion
Datadog integration is available only for M10+ clusters.`,
```